### PR TITLE
Updated Korean Translation.

### PR DESCRIPTION
**Pull request by steamb23**
Not complete.
Please, Don't delete 'SteamB23's Korean Translation Bookmark' in 485 line.

### CODE REVIEW

**Comment by Freso** [comment]
Do you want us to wait with merging this in until you've completed it?

File: masterlist.yaml [hunk 1]
```diff
@@ -67,6 +75,8 @@ globals:
         str: 'Ваш SKSE имеет последнюю версию.'
       - lang: es
         str: 'Tienes la última versión de SKSE'
+      - lang:
```
Comment (Freso) [inline masterlist.yaml]: Missing `ko` here.

File: masterlist.yaml [hunk 2]
```diff
@@ -36282,3 +36311,4 @@ plugins:
       - type: warn
         condition: 'active("MoonAndStar_MAS.esp") and not active("MoonAndStar_Undeath_compat.esp")'
         content: 'This mod has certain conflicts with "MoonAndStar_MAS.esp". Consider using [the compatibility patch](http://www.nexusmods.com/skyrim/mods/52397/).'
+
```
Comment (Freso) [inline masterlist.yaml]: Why did add a blank line?

**Comment by steamb23** [comment]
oh..
Sorry.
I work manually, it is a mistake a lot.
A little more work, and I'll fix and then again pull request.

**Comment by steamb23** [comment]
I don't know why that additional blank lines.

**Comment by WrinklyNinja** [comment]
Don't close this PR: just push your fixes, and they'll be added to it.

**Comment by steamb23** [comment]
Oh, also to do so. now I know.

**Comment by steamb23** [comment]
I don't know how to resolve the merge conflict.
I'm back from the beginning.

**Comment by Freso** [comment]
I'm just gonna pull this in then, and you can continue working from upstream's master. ;) Consider making a branch for future updates, so you don't have to worry with changes to the master branch in the main repository. :)

**Comment by steamb23** [comment]
how can I use the single quotation marks? Can I use an escape character?
(The Korean sentence does not use double quotes to emphasize a word. Only be used in the quote or dialogue.)

**Comment by Freso** [comment]
> how can I use the single quotation marks?

By using double single quotation marks, e.g., `This mod has certain conflicts with ''MoonAndStar_MAS.esp''.` will give "This mod has certain conflicts with 'MoonAndStar_MAS.esp'.". It is not specified directly in https://loot.github.io/docs/dev/LOOT Metadata Syntax.html, but there are a few examples of it being used.

**Comment by Freso** [comment]
Another possibility would be to use Unicode single quotation marks, `‘` and `’`, of course, which also carry a better semantic meaning.

**Pull request merged**